### PR TITLE
Add keybindings to readme #33

### PR DESCRIPTION
**Pull request by kjense54**
Added 4 tables (navigation, image, container and volume) to the readme under a Keybinds section. Added Keybinds to the content list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +89,53 @@ Now, **goManageDocker 😏!!**
    
    ![search](vhs/gifs/search.gif)
 
+## Keybinds
+
+### Navigation
+| Operation | Key                                                                 |
+|-----------|---------------------------------------------------------------------|
+| Enter     | <kbd>Enter</kbd>                                                    |
```
Comment (ajayd-san) [inline README.md:97]: You may omit `Enter`

File: README.md [hunk 2]
```diff
@@ -88,6 +89,53 @@ Now, **goManageDocker 😏!!**
...
+| Next Item | <kbd>↓</kbd> / <kbd>j</kbd>                                         |
+| Prev Item | <kbd>↑</kbd> / <kbd>k</kbd>                                         |
+| Next Page | <kbd>[</kbd>                                                        |
+| Prev Page | <kbd>]</kbd>                                                        |
+
+### Image
+| Operation         | Key                                                           |
+|-------------------|---------------------------------------------------------------|
+| Run               | <kbd>r</kbd>                                                  |
+| Rename            | <kbd>R</kbd>                                                  |
```
Comment (ajayd-san) [inline README.md:111]: Omit `Rename` entry since we do not support that operation yet.

File: README.md [hunk 3]
```diff
@@ -88,6 +89,53 @@ Now, **goManageDocker 😏!!**
...
+| Scout             | <kbd>s</kbd>                                                  |
+| Prune             | <kbd>p</kbd>                                                  |
+| Delete            | <kbd>d</kbd>                                                  |
+| Delete Force      | <kbd>D</kbd>                                                  |
+| Copy Id           | <kbd>c</kbd>                                                  |
```
Comment (ajayd-san) [inline README.md:116]:
```suggestion
| Copy ID           | <kbd>c</kbd>                                                  |
```

File: README.md [hunk 4]
```diff
@@ -88,6 +89,53 @@ Now, **goManageDocker 😏!!**
...
+### Container
+| Operation         | Key                                                           |
+|-------------------|---------------------------------------------------------------|
+| Toggle List All   | <kbd>a</kbd>                                                  |
+| Toggle Start/Stop | <kbd>s</kbd>                                                  |
+| Toggle Pause      | <kbd>t</kbd>                                                  |
+| Restart           | <kbd>r</kbd>                                                  |
+| Delete            | <kbd>d</kbd>                                                  |
+| Delete Force      | <kbd>D</kbd>                                                  |
+| Execute           | <kbd>x</kbd>                                                  |
```
Comment (ajayd-san) [inline README.md:128]:
```suggestion
| Exec          | <kbd>x</kbd>                                                  |
```

File: README.md [hunk 5]
```diff
@@ -88,6 +89,53 @@ Now, **goManageDocker 😏!!**
...
+| Prune             | <kbd>p</kbd>                                                  |
+| Copy Id           | <kbd>c</kbd>                                                  |
+
+### Volume
+| Operation         | Key                                                           |
+|-------------------|---------------------------------------------------------------|
+| Delete            | <kbd>d</kbd>                                                  |
+| Prune             | <kbd>p</kbd>                                                  |
+| Copy Name         | <kbd>c</kbd>                                                  |
```
Comment (ajayd-san) [inline README.md:137]:
```suggestion
| Copy Volume Name         | <kbd>c</kbd>                                                  |
```

File: README.md [hunk 6]
```diff
@@ -88,6 +89,53 @@ Now, **goManageDocker 😏!!**
...
+|-------------------|---------------------------------------------------------------|
+| Toggle List All   | <kbd>a</kbd>                                                  |
+| Toggle Start/Stop | <kbd>s</kbd>                                                  |
+| Toggle Pause      | <kbd>t</kbd>                                                  |
+| Restart           | <kbd>r</kbd>                                                  |
+| Delete            | <kbd>d</kbd>                                                  |
+| Delete Force      | <kbd>D</kbd>                                                  |
+| Execute           | <kbd>x</kbd>                                                  |
+| Prune             | <kbd>p</kbd>                                                  |
+| Copy Id           | <kbd>c</kbd>                                                  |
```
Comment (ajayd-san) [inline README.md:130]:
```suggestion
| Copy ID           | <kbd>c</kbd>                                                  |
```

**Review by ajayd-san — CHANGES_REQUESTED** [review]
Hey! Thanks for the PR!!

I have a few suggestions, please do them and revert back. Thanks!

File: README.md (hunk 4 above)
Comment (kjense54) [reply README.md]: Should "Run and Execute" also be "Run and Exec" or stay as is

Comment (ajayd-san) [reply README.md]: Yes! Please change that as well.

File: README.md [hunk 7]
```diff
@@ -88,6 +89,51 @@ Now, **goManageDocker 😏!!**
...
+| Prev Page | <kbd>]</kbd>                                                        |
+
+### Image
+| Operation         | Key                                                           |
+|-------------------|---------------------------------------------------------------|
+| Run               | <kbd>r</kbd>                                                  |
+| Scout             | <kbd>s</kbd>                                                  |
+| Prune             | <kbd>p</kbd>                                                  |
+| Delete            | <kbd>d</kbd>                                                  |
+| Delete Force      | <kbd>D</kbd>                                                  |
```
Comment (ajayd-san) [inline README.md:113]:
```suggestion
| Delete (Force)      | <kbd>D</kbd>                                                  |
```

File: README.md [hunk 8]
```diff
@@ -88,6 +89,51 @@ Now, **goManageDocker 😏!!**
...
+
+### Container
+| Operation         | Key                                                           |
+|-------------------|---------------------------------------------------------------|
+| Toggle List All   | <kbd>a</kbd>                                                  |
+| Toggle Start/Stop | <kbd>s</kbd>                                                  |
+| Toggle Pause      | <kbd>t</kbd>                                                  |
+| Restart           | <kbd>r</kbd>                                                  |
+| Delete            | <kbd>d</kbd>                                                  |
+| Delete Force      | <kbd>D</kbd>                                                  |
```
Comment (ajayd-san) [inline README.md:125]:
```suggestion
| Delete (Force)      | <kbd>D</kbd>                                                  |
```

**Review by ajayd-san — CHANGES_REQUESTED** [review]
Hey! Can you do those 2 final suggestions, I'll merge to main tonight.

Thanks!

**Comment by ajayd-san** [comment]
that's sick, thanks!

**Pull request merged**